backend/models/user.js

```diff
@@ -153,7 +153,7 @@ User.authenticate =  async function (logger, username, password) {
 
 	let user = null;
 
-	if (C.EMAIL_REGEXP.test(username)) { // if the submited username is the email
+	if (await utils.isEmail(username)) { // if the submited username is the email
```

Comment:
if we're using yup, it's better to just have the schema validation of the request instead of using  yup as part of the utils.